src/ReleaseHistory.md

```diff
@@ -347,3 +347,4 @@
 * API NON-BREAKING: Add suppression.justification. https://github.com/oasis-tcs/sarif-spec/issues/373
 * API NON-BREAKING: Associate descriptor metadata with thread flow locations. https://github.com/oasis-tcs/sarif-spec/issues/381
 * API BREAKING: Move `location.physicalLocation.id` to `location.id`. https://github.com/oasis-tcs/sarif-spec/issues/375
+* API BREAKING: Rename artifact role "traceFile" to "tracedFile". Add artifact role "debugOutputFile".
```

Comment:
>" [](start = 37, length = 1)

use markdown back ticks to enclose these names, not double quotes. :)

Reply:
then merge!

---
In reply to: [277865223](https://github.com/Microsoft/sarif-sdk/pull/1447#discussion_r277865223) [](ancestors = 277865223)

Reply:
oops! thanks for catching this :)